Gems/Multiplayer/Code/Source/MultiplayerSystemComponent.cpp

```diff
@@ -748,6 +748,17 @@ namespace Multiplayer
     {
         reinterpret_cast<ServerToClientConnectionData*>(connection->GetUserData())->SetProviderTicket(packet.GetTicket().c_str());
 
+        const char* levelName = AZ::Interface<AzFramework::ILevelSystemLifecycle>::Get()->GetCurrentLevelName();
+        if (!levelName || strlen(levelName) == 0)
```

Comment:
You probably don't even need `strlen` if you dont care about trimming or spaces:
```suggestion
        if (!levelName || *levelName == '\0')
```